fhir-persistence-schema/docs/DropTenant.md

```diff
@@ -0,0 +1,129 @@
+# Removing a tenant from the database
+
+Use this guide when removing a tenant from the Db2 multi-tenant database. The tenant's data will be removed from the database along with any associated storage.
+
+To obtain a list of the tenants currently managed in a given database, run the following:
+
+```sh
+    java -jar schema/fhir-persistence-schema-*-cli.jar \
+      --prop-file db2.properties --list-tenants
+```
+
+```
+ TenantId     Status       TenantName Schema
+        1  ALLOCATED            test1 FHIRDATA
+        2  ALLOCATED            test2 FHIRDATA2
+        3    DROPPED            test3 <not-known>
+```
```

Comment:
<not-known> because it's created by old version of the tool which didn't record the schema name?
if so, could we make a tool (not in this PR) to check the system table just as you showed me before to fix those <not-known>?